mongoengine/fields.py

```diff
@@ -198,6 +199,21 @@ def validate(self, value):
             self.error('Float value is too large')
 
 
+class DecimalField(BaseField):
+    """A decimal field.
+
+    This uses strings to store the value in MongoDB.
```

Comment:
I'd add a note explaining why storing as strings is preferable to storing as floats.